### PR TITLE
Multi3Generation Ingestion

### PR DESCRIPTION
**Pull request by anthology-assist**

### CODE REVIEW

File: data/xml/2023.multi3generation.xml [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+      <editor><first>Anabela</first><last>Barreiro</last></editor>
+      <editor><first>Max</first><last>Silberztein</last></editor>
+      <editor><first>Elena</first><last>Lloret</last></editor>
+      <editor><first>Marcin</first><last>Paprzycki</last></editor>
+      <publisher>European Association for Machine Translation</publisher>
+      <address>Tampere, Finland</address>
+      <month>June</month>
+      <year>2023</year>
+      <url hash="1d6576c8">2023.multi3generation-1</url>
+      <venue>multi3generation</venue>
```
Comment (mjpost) [inline data/xml/2023.multi3generation.xml:15]: Please add ws venue

File: data/yaml/venues/multi3generation.yaml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+acronym: Multi3Generation
+name: The 1st International Workshop on Multilingual, Multimodal and Multitask Language
```
Comment (mjpost) [inline data/yaml/venues/multi3generation.yaml:2]: The name in the venues file is the name of the workshop, not a meeting of the workshop (so no "1st")

**Review by mjpost — COMMENTED** [review]
Is there a corresponding issue that should can be linked in the sidebar?

**Comment by mjpost** [comment]
Please link in sidebar ("Development" header): https://github.com/acl-org/acl-anthology/issues/2892

**Comment by kjp-souza** [comment]
> Please link in sidebar ("Development" header): #2892

Hi @mjpost - are any of these actions required from our side? Do we need to change the metadata file? I'm a bit in doubt. Thanks!

**Comment by mjpost** [comment]
@kjp-souza No these are tasks on our end

File: data/yaml/venues/multi3generation.yaml (hunk 2 above)
Comment (anthology-assist) [reply data/yaml/venues/multi3generation.yaml]: Done.

File: data/xml/2023.multi3generation.xml (hunk 1 above)
Comment (anthology-assist) [reply data/xml/2023.multi3generation.xml:15]: Done.

**Pull request merged**